cypress/integration/purchase/purchase_order_complete_reactivate_complete.js

```diff
@@ -0,0 +1,167 @@
+        .setValue(productCategoryValue)
+        .apply();
+    });
+
+    Builder.createBasicProductEntitiesWithCUTUAllocation(
+      productCategoryName,
+      productCategoryValue,
+      priceListName,
+      productName1,
+      productValue1,
+      productType,
+      packingInstructionsName
+    );
+    Builder.createBasicProductEntitiesWithCUTUAllocation(
+      productCategoryName,
+      productCategoryValue,
+      priceListName,
+      productName2,
+      productValue2,
+      productType,
+      packingInstructionsName
+    );
+    cy.fixture('sales/simple_vendor.json').then(vendorJson => {
+      new BPartner({ name: vendorName })
+        .setVendor(true)
+        .setVendorPricingSystem(priceSystemName)
+        .setVendorDiscountSchema(discountSchemaName)
+        .setPaymentTerm('30 days net')
+        .addLocation(new BPartnerLocation('Address1').setCity('Cologne').setCountry('Deutschland'))
+        .apply();
+    });
+    cy.readAllNotifications();
+  });
+  it('Create a purchase order', function() {
+    cy.visitWindow('181', 'NEW');
+    cy.get('#lookup_C_BPartner_ID input')
+      .type(vendorName)
+      .type('\n');
+    cy.contains('.input-dropdown-list-option', vendorName).click();
+
+    cy.selectInListField('M_PricingSystem_ID', priceSystemName, false, null, true);
+    const addNewText = Cypress.messages.window.batchEntry.caption;
+    cy.get('.tabs-wrapper .form-flex-align .btn')
+      .contains(addNewText)
+      .should('exist')
+      .click();
+    // cy.wait(8000);
+    cy.get('.quick-input-container .form-group').should('exist');
+    cy.writeIntoLookupListField('M_Product_ID', productName1, productName1, false, false, null, true);
+
+    cy.get('.form-field-Qty')
+      .click()
+      .find('.input-body-container.focused')
+      .should('exist')
+      .find('i')
+      .eq(0)
+      .click();
+    cy.server();
+    cy.route('POST', `/rest/api/window/${purchaseOrders.windowId}/*/${purchaseOrders.orderLineTabId}/quickInput`).as(
+      'resetQuickInputFields'
+    );
+    cy.get('.form-field-Qty')
+      .find('input')
+      .should('have.value', '0.1')
+      .clear()
+      .type('5{enter}');
+    cy.wait(8000);
+
+    /**Complete purchase order */
+    cy.get('.form-field-DocAction ul')
+      .click({ force: true })
+      .get('li')
+      .eq('1')
+      .click({ force: true });
+    cy.wait(8000);
+    /**check product name */
+    cy.get('tbody tr')
+      .eq('0')
+      .find('.Lookup')
+      .find('.lookup-cell')
+      .contains(productName1);
+    /**check price of product */
+    cy.get('tbody tr')
+      .eq('0')
+      .find('.CostPrice')
+      .find('.costprice-cell')
+      .eq(0)
+      .contains('1.23');
+    /**check product quantity */
+    cy.get('tbody tr')
+      .eq('0')
+      .find('.quantity-cell')
+      .contains('5');
+    /**purchase order should be completed */
+    cy.log('purchase order should be completed');
+    cy.get('.tag.tag-success').contains('Completed');
```

Comment:
don'T we need to use the dictionary or something, to be language independent?

Reply:
you are right, I'll check:)